x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/header.tsx

```diff
@@ -51,13 +51,23 @@ const FlyoutHeaderComponent: React.FC<Props> = ({
                 <EuiFlexItem grow={false}>
                   <EuiTitle size="s">
                     <h3 id="flyoutTitle">
-                      <FormattedMessage
-                        defaultMessage="{actionTypeName} connector"
-                        id="xpack.triggersActionsUI.sections.addConnectorForm.flyoutTitle"
-                        values={{
-                          actionTypeName,
-                        }}
-                      />
+                      {actionTypeName && actionTypeName.toLowerCase().includes('connector') ? (
+                        <FormattedMessage
+                          defaultMessage="{actionTypeName}"
+                          id="xpack.triggersActionsUI.sections.addConnectorForm.flyoutTitle"
+                          values={{
+                            actionTypeName,
+                          }}
+                        />
+                      ) : (
+                        <FormattedMessage
+                          defaultMessage="{actionTypeName} connector"
+                          id="xpack.triggersActionsUI.sections.addConnectorForm.flyoutTitle"
+                          values={{
+                            actionTypeName,
```

Comment:
Shouldn't use the same message id for different messages.

(also I think we shouldn't be using i18n for something that has a default message that is only a variable. `{actionTypeName}` will translate to `{actionTypeName}` in any language)